src/ui/component/app/view.jsx

```diff
@@ -65,10 +69,22 @@ function App(props: Props) {
   }, [theme]);
 
   useEffect(() => {
-    if (userId) {
+    if (previousUserId === undefined && userId) {
```

Comment:
`=== undefined`? or just `!previousUserId` ?

Reply:
same question for below `undefined` checks

Reply:
I guess this one specifically could be just be `!previousUserId` but the other ones need to check that they weren't `undefined` so the event isn't fired from the initial user fetch